tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
so just to be clear, what you are proposing is that every class (except those rare ones that have neither move nor copy semantics) we write will have to have equivalence and inequivalence operators...